client/common/apollo.tsx

```diff
@@ -36,8 +37,37 @@ const getHttpLink = (headers?: IncomingHttpHeaders) =>
     headers,
   });
 
+const indexedDbLink = new ApolloLink((operation) => {
+  console.log({ operation });
+
+  return new Observable((observer) => {
+    observer.next({
+      data: {
+        items: {
+          edges: [],
+          pageInfo: {
+            endCursor: 'YXJyYXljb25uZWN0aW9uOjIz',
+            hasNextPage: true,
+            __typename: 'PageInfo',
+          },
+          __typename: 'ItemConnection',
+        },
+        itemSuggestions: [],
+      },
+    });
+    observer.complete();
+  });
```

Comment:
we will need to replace this with our own client-side resolvers, one for the items query, one for the sets query, etc